src/ServiceBus.AttachmentPlugin/ServiceBus.AttachmentPlugin.csproj

```diff
@@ -5,9 +5,12 @@
     <Version>6.0.0-preview01</Version>
     <Authors>Sean Feldman</Authors>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
these 3 lines should be in your directory props file

Reply:
`TreatWarningsAsErrors` and `LangVersion` were already there 😄 
Fixing